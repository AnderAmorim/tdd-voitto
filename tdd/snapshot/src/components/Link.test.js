import React from 'react'
import renderer from 'react-test-renderer'
import Link from './Link'

test('Verificamdo conteÃºdo de Link.js', () => {
    const app = renderer.create(
        <Link page="http://www.facebook.com.br">Facebookkkkk</Link>
    ).toJSON()
    expect(app).toMatchSnapshot()
})