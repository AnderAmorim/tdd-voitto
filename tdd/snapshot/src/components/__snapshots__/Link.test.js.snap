// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Verificamdo conte√∫do de Link.js 1`] = `
<a
  className="normal"
  href="http://www.facebook.com.br"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  Facebookkkkk
</a>
`;
